notifiers/providers/victorops.py

```diff
@@ -0,0 +1,78 @@
+from ..core import Provider
+from ..core import Response
+from ..utils import requests
+
+
+class VictorOps(Provider):
+    """Send VictorOps webhook notifications"""
+
+    base_url = "https://portal.victorops.com/ui/{ORGANIZATION_ID}/incidents"
+    site_url = "https://portal.victorops.com/dash/{ORGANIZATION_ID}#/advanced/rest"
+    name = "victorops"
+
+    _required = {
+        "required": [
+            "rest_url",
+            "message_type",
+            "entity_id",
+            "entity_display_name",
+            "message",
+        ]
+    }
+    _schema = {
+        "type": "object",
+        "properties": {
+            "rest_url": {
+                "type": "string",
+                "format": "uri",
+                "title": "the REST URL to use with routing_key. create one in victorops `integrations` tab.",
+            },
+            "message_type": {
+                "type": "string",
+                "title": "severity level can be: "
+                "- CRITICAL or WARNING: Triggers an incident "
+                "- ACKNOWLEDGEMENT: sends Acknowledgment to an incident "
+                "- INFO: Creates a timeline event but doesn't trigger an incident "
+                "- RECOVERY: Resolves an incident",
+                "enum": ["CRITICAL", "WARNING", "ACKNOWLEDGEMENT", "INFO", "RECOVERY"],
```

Comment:
I rather have the enum as lowercase and then uppercasing it when preparing the request (if that's mandatory)